Formula/rocketmq.rb

```diff
@@ -0,0 +1,133 @@
+class Rocketmq < Formula
+  desc "Distributed messaging and streaming platform"
+  homepage "https://rocketmq.apache.org"
+  url "https://www.apache.org/dyn/closer.lua?path=rocketmq/4.9.0/rocketmq-all-4.9.0-bin-release.zip"
+  mirror "https://archive.apache.org/dist/rocketmq/4.9.0/rocketmq-all-4.9.0-bin-release.zip"
```

Comment:
Does this include any precompiled binaries?

Reply:
It only includes precompiled Java cross-platform binaries.